### ISSUE TITLE
Broken relative paths in `bpmn-js/dist/assets/bpmn-font/bpmn-embedded.css`

### ISSUE THREAD

**Issue by MLNW** [bpmn-io/bpmn-js#1976]
### Describe the Bug

Since a recent release the file `bpmn-js/dist/assets/bpmn-font/bpmn-embedded.css` (which seems to be generated) has been moved, thereby breaking some of these relative paths:

```css
@font-face {
  ...
  src: url('../font/bpmn.eot?16406289');
  src: url('../font/bpmn.eot?16406289#iefix') format('embedded-opentype'),
       url('../font/bpmn.svg?16406289#bpmn') format('svg');
  ...
}
```

Due to the change I get errors like this:

```console
../font/bpmn.eot?16406289 referenced in /projects/bpmn/src/app.postcss didn't resolve at build time, it will remain unchanged to be resolved at runtime
../font/bpmn.eot?16406289#iefix referenced in /projects/bpmn/src/app.postcss didn't resolve at build time, it will remain unchanged to be resolved at runtime
../font/bpmn.svg?16406289#bpmn referenced in /projects/bpmn/src/app.postcss didn't resolve at build time, it will remain unchanged to be resolved at runtime
```

### Expected Behavior

I think the URLs should be like this:

```css
@font-face {
  ...
  src: url('./bpmn.eot?16406289');
  src: url('./bpmn.eot?16406289#iefix') format('embedded-opentype'),
       url('./bpmn.svg?16406289#bpmn') format('svg');
  ...
}
```

As the folder structure in my node_modules looks like this:

![grafik](https://github.com/bpmn-io/bpmn-js/assets/20930814/f4cfa491-e792-4b52-8c0e-d399c6bcdde9)

### Environment

- Library version: 14.1.0

**Comment by nikku** [comment]
Which release did work for you (last)?

**Comment by nikku** [comment]
I can confirm this broke from [14.0.0](https://unpkg.com/browse/bpmn-js@14.0.0/dist/assets/) to [14.1.0](https://unpkg.com/browse/bpmn-js@14.1.0/dist/assets/).

**Comment by nikku** [comment]
Will be addressed via https://github.com/bpmn-io/bpmn-js/pull/1977 shortly.

**Comment by barmac** [comment]
Closed via #1977 and already released as v14.1.1.
